shared/unattended/Fedora-24.ks

```diff
@@ -0,0 +1,40 @@
+install
+KVM_TEST_MEDIUM
+text
```

Comment:
Humm... I see that these are only on EL-7 based kickstart files, and I copied this from Fedora-23.  TBH I was not aware of `GRAPHICAL_OR_TEXT` and was even confused for a while... so much that I created:

https://trello.com/c/ox7NdekL/749-avocado-vt-add-proper-template-prefix-for-kickstart-files

Anyway, updating the KS.